### DISCUSSION TITLE
How to control language (script) rendering in citations?

### DISCUSSION THREAD

**Discussion by nikola-grubor** [quarto-dev/quarto-cli#6449] [Q&A] [answered]
### Description

I have set the following language options:
```
---
title: "Example"
lang: sr-latn
---
```
However the automatic citation renderer translates linking words like "and" into Cyrillic script, printing "и" instead of the proper "i". 
sr-latn should be a valid language code and script, as the official Serbian language supports both scripts (Latin and Cyrillic).

Is there a way to manually control script rendering for dates, citations, etc.?

**Comment by mcanouil** [comment]
Could you share a small self-contained "working" (reproducible) example to work with, _i.e._, a complete Quarto document or a Git repository? Thanks.

This being said, CSL files/templates are supposed to take care of the translation as long as they do have the translation (<https://docs.citationstyles.org/en/stable/translating-locale-files.html?highlight=language>).

You can share a Quarto document using the following syntax, _i.e._, using more backticks than you have in your document (usually four ` ```` `).

`````md
````qmd
---
title: "Reproducible Quarto Document"
format: html
---

This is a reproducible Quarto document using `format: html`.
It is written in Markdown and contains embedded R code.
When you run the code, it will produce a plot.

```{r}
plot(cars)
```

The end.
````
`````

**Reply by nikola-grubor** [reply]
If Quarto's automatic citation engine takes on translations from [github.com/citation-style-language/locales](https://github.com/citation-style-language/locales) then it seems to be a problem of a missing proper translation/locale file. I'll work on one to submit. Thanks!

**Reply by mcanouil** [reply]
To note, the citation system is actually the one from Pandoc.

**Reply by nikola-grubor** [reply]
So after trying different things I've decided to add a [reproducible example repository](https://github.com/nikola-grubor/quarto-locale-example).

Looking at the [BCP 47](https://www.rfc-editor.org/info/rfc5646) "sr-Latn" should be a way to specify the appropriate script. I've been trying to find from where does Pandoc get its [translations](https://pandoc.org/MANUAL.html#language-variables) so that I can submit an appropriate file if one does not exist, but I'm having a hard time figuring it out. The Babel package (used by Pandoc) seems to have the [required translations](https://github.com/latex3/babel/tree/main/locale/sr), but these are used only in case of pdf generation?

**Reply by dragonstyle** [reply, marked as answer]
Localizations for citation processing (which uses `citeproc`) is here:

https://github.com/jgm/citeproc/tree/master/locales

It looks like there isn't a `sr-Latn` specific locale. I'm sure it would be very much appreciated if you could submit one!
